Sources/SwiftParser/Lookahead.swift

```diff
@@ -207,42 +207,35 @@ extension Parser.Lookahead {
     return true
   }
 
+  /// Tries consuming a `#if` directive that only contains attributes.
+  /// Returns `true` if that was successful and `false` if
+  ///  - we are not at a valid `#if` directive (e.g. `#elseif` is missing)
+  ///  - the directive contained non-attributes
+  ///  - the directive did not contain any attributes
   mutating func consumeIfConfigOfAttributes() -> Bool {
-    while true {
-      // #if / #else / #elseif
-      self.consumeAnyToken()
+    var didSeeAnyAttributes = false
+    var poundIfLoopProgress = LoopProgressCondition()
+    repeat {
+      self.consume(ifAny: [.poundIfKeyword, .poundElseKeyword, .poundEndifKeyword])
```

Comment:
You meant `poundElseifKeyword`?
```suggestion
      assert(self.at(any: [.poundIfKeyword, .poundElseifKeyword, .poundElseKeyword])
      consumeAnyToken()
```

Reply:
Yes, thanks